portfolio/src/main/webapp/index.html

```diff
@@ -2,16 +2,93 @@
+        and working at Apple Retail as a tech expert. High school was an incredible learning experience
+        for me, and I am greatful that I was able to find my passions so early on. However, even 
+        though working at these companies gives me incredible joy, I have found true fullfillment when
+        starting my own things. That's why I always have a side project that I am working on, something to
+        dream with.
+        <br> <br>
+        Now I am currently a sophomore at Duke University studying Computer Science and Physics. This past 
+        summer I just finished my most recent internship with Facebook doing mobile development as part of 
+        their Freshman/Sophomore internship program. In between classes I do machine learning research 
+        developing automated fact checking solutions with my Databases Professor, Jun Yang. I am involved 
+        with Chi Psi and DSP, social and professional fraternities I am a part of. I am a part of Duke's 
+        Student Founders Program working on developing some of the startup ideas I have into companies. And 
+        I am currently working on writing a sci-fi novel as part of a publishing program run by the Creator 
+        Institute.
+        <br> <br>
+        Aside, from my obsessions with work, startups, and tech I have found several pastimes
+        that I fill the freetime I have with. I love playing chess wtih anyone who is willing to play. I've 
+        played soccer since I was 5 years old and, even though I don't play as much anymore, I would never give 
+        up the chance to play another rec game. I'm an aspiring ping pong player who is more bark than bite.
+        And I'm obsessed with sci-fi, movies and novels alike. I still get chills when I think of Interstellar or
+        the Three Body Trilogy. 
+      </p>
+      <h2>My Projects</h2>
+      <ul>
+          <li>
+            <a href="https://caction.adamgonen.now.sh/">Climate Action</a>,
+            a website that analyzes how climate change has developed over time 
+            with thousands of datapoints stored in the cloud that shows a historical
+            trend from the 1900s to now.
+          </li>
+          <li>
+            <a href="https://github.com/AndresM77/ReConnect">Reconnect</a>, 
+            a networking app that allows you to maintian the connections you have, 
+            set up meetings with them, get notified when they are in your location,
+            sort your contacts by industry, and keep track of how frequently 
+            you have been keeping up with them.
+          </li>
+      </ul>
+      <h2>On the Internet:</h2>
+      <ul>
+          <li>
+                <a href="https://www.linkedin.com/in/andresm77/">LinkedIn</a>
+          </li>
+          <li>
+                <a href="https://github.com/AndresM77">GitHub</a>
+          </li>
+      </ul>
+      <h2>Contact Me:</h2>
+      <ul>
+          <li>
+              Email: andres@triplean.com
```

Comment:
You can make this a clickable mail link with a `mailto:`

`<a href="mailto:andres@triplean.com">andres@triplean.com</a>`